pkg/lib/operation/project/remote/file/download/operation.go

```diff
@@ -0,0 +1,101 @@
+// Package download contains the implementation of the "kbc project remote file download" command.
+package download
+
+import (
+	"context"
+	"io"
+	"os"
+	"sync"
+
+	"github.com/keboola/go-client/pkg/keboola"
+	"go.opentelemetry.io/otel/trace"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/log"
+	"github.com/keboola/keboola-as-code/internal/pkg/telemetry"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+type dependencies interface {
+	KeboolaProjectAPI() *keboola.API
+	Logger() log.Logger
+	Tracer() trace.Tracer
+}
+
+type Options struct {
+	File   *keboola.FileDownloadCredentials
+	Output string
+}
+
+func Run(ctx context.Context, opts Options, d dependencies) (err error) {
+	ctx, span := d.Tracer().Start(ctx, "kac.lib.operation.project.remote.file.download")
+	defer telemetry.EndSpan(span, &err)
+
+	if opts.File.IsSliced {
+		if opts.Output != "-" {
+			err = os.Mkdir(opts.Output, 0o755)  // nolint: forbidigo
+			if err != nil && !os.IsExist(err) { // nolint: forbidigo
+				return errors.Errorf("cannot create directory %s: %w", opts.Output, err)
+			}
+		}
+
+		slices, err := keboola.DownloadManifest(ctx, opts.File)
+		stdoutLock := &sync.Mutex{}
+		for _, slice := range slices {
```

Comment:
Skor bude asi potrebne skontrolovat, ci sa slices vratia v poradi z API/go-client.
Pripadne ich mozno musime zoradit my, mala by tam byt taka logika, ze je to `slice_001`, `slice_002` ....

Reply:
Á, tak už asi vím. V jsonu v `initial-state` jsou slajsy jako objekt, tam se to pořadí nejspíš zamíchá. 

https://github.com/keboola/keboola-as-code/blob/6ec30cd751c9d55107b163634c4cdb8b1e535901/internal/pkg/utils/testproject/project.go#L346-L347

Nejspíš z toho teda udělám pole s v nořenýma objektama:
```
  "slices": [
    {
      "name": "slice1",
      "content": "val1,val2\nval3,val4"
    },
  ]
```